imap_processing/lo/l0/data_classes/science_counts.py

```diff
@@ -0,0 +1,285 @@
+"""L1A Science Counts data class."""
+from dataclasses import dataclass
+
+import numpy as np
+from bitstring import ConstBitStream
+
+from imap_processing.ccsds.ccsds_data import CcsdsData
+from imap_processing.lo.l0.utils.bit_decompression import Decompress, decompress_int
+from imap_processing.lo.l0.utils.lo_base import LoBase
+
+
+@dataclass
+class ScienceCounts(LoBase):
+    """L1A Science Count data class.
+
+    The Science Counts class handles the parsing
+    and decompression of L0 to L1A data.
+
+    Attributes
+    ----------
+    SHCOARSE: int
```

Comment:
All of these should have a space before the colon I think. I wonder if that was causing issues with the doc build.
https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring
```suggestion
    SHCOARSE : int
```